tests/foreman/ui/test_host.py

```diff
@@ -1428,6 +1428,79 @@ def test_positive_global_registration_end_to_end(
         assert 'execution' in interface[0]['type']
 
 
+@pytest.mark.stubbed
+@pytest.mark.tier2
+def test_global_registration_form_populate(session):
+    """Host registration form should be get populate automatically based on the host-group
+
+    :id: b949e010-36b8-48b8-9907-36138342c72b
+
+    :expectedresults: Some of the fields in the form should be populated based on host-group
```

Comment:
Which values in the form will be populated based on the host group? Can you list them in `:expectedresults:`?